src/CONST.ts

```diff
@@ -1742,6 +1742,11 @@ const CONST = {
             DISABLE: 'disable',
             ENABLE: 'enable',
         },
+        REPORT_FIELDS_VALUES_BULK_ACTION_TYPES: {
+            DELETE: 'delete',
+            DISABLE: 'disable',
+            ENABLE: 'enable',
+        },
```

Comment:
I feel like we would want to merge these together to some generic BULK_ACTION_TYPE eventuallly

Reply:
So, I am going to do it in a scope of this PR.

Reply:
Done - 95856f1d8b804d02a3ec46c936bb8a08a8083da5.